src/site/stages/build/plugins/create-resources-and-support-section.js

```diff
@@ -326,15 +325,6 @@ function createSearchResults(files) {
   };
 }
 
-// @todo remove this to enable indexing
-function excludeFromSiteMap(files) {
-  const allArticles = getArticlesBelongingToResourcesAndSupportSection(files);
-
-  allArticles.forEach(article => {
-    article.private = true;
-  });
-}
-
```

Comment:
We have to remove the function call `excludeFromSiteMap(files)` a few lines down

Reply:
Oh perfect! Thanks! I appreciate you giving me the test below as well!